iamclient/iamclient.go

```diff
@@ -141,6 +162,51 @@ func (client *Client) GetSystemID() (systemID string) {
 	return client.systemID
 }
 
+// GetNodeInfo returns node info.
+func (client *Client) GetNodeInfo(nodeID string) (nodeInfo *pb.NodeInfo, err error) {
```

Comment:
No PB types in public API. It should return `cloudprotocol.NodeInfo`. I will add conversion function for aos_common pvconvert.

Reply:
done